test/simple_cromwell.conf

```diff
@@ -0,0 +1,71 @@
+include required(classpath("application"))
+
+engine {
+  filesystems {
+    gcs {
+      auth = "application-default"
+      }
+  }
+}
+
+backend.providers.Local.config.filesystems {
+	gcs {
+		auth = "application-default"
+	}
+}
+
+backend {
+  default = GCPBATCH
+
+  providers {
+    GCPBATCH {
+      actor-factory = "cromwell.backend.google.batch.GcpBatchBackendLifecycleActorFactory"
+      config {
+        project = "broad-dsde-methods"
+
+        root = "gs://palantir-workflows-test-execution"
+
+        maximum-polling-interval = 600
+
+        genomics {
+          auth = "application-default"
+
+          location = "us-central1"
+
+          parallel-composite-upload-threshold="150M"
+        }
+
+        filesystems {
+          gcs {
+            auth = "application-default"
+            project = "broad-dsde-methods"
+
+            caching {
```

Comment:
For caching, this will look at the `test-execution` bucket for previous runs of the test, correct? Is there a way to make sure that bucket stays "clean" with just one copy to call-cache from and not build up a bunch of clutter over time from each PR? Does it have some TTL policy to help with this?

Reply:
I think there are a few options here, though I don't think any of them are perfect.  We could set `duplication-strategy` to `reference`, which would reduce call-caching related clutter, however would not play well with TTL, since call-caching would break every few months when the bucket was cleaned up.  At the moment, I'm leaving this as `copy`, but with TTL of 90 days, so things will get cluttered, but there will be a consistent cleanup as well.  I think once we see how much data actually sits in this bucket at steady state we can figure out the right settings.

The other thing to note is that call-caching is "on" in name only currently, since I haven't actually set up an external database to cache from.  I think that could be a nice addition in the future, but again I think it's worth getting an idea what the compute costs associated with this testing actually are, since a cloud sql instance on its own looks like it would cost $10-20 per day.  Would be nice for the speedup, but it's possible we should think of it more as a time saver than a money saver.

Reply:
$10-$20 per day still seems like a lot for a repo that doesn't have that many PRs per day typically. Does the `copy` setting just look inside the bucket for a previous run and then cache based on that? I think doing a pilot with 90 days TTL sounds reasonable to get started.

Reply:
The cache lookup is based on a database, which is either in memory, or an external database.  When the database in in memory (as in this default case), it essentially means that it can't access info about any workflows from previous runs of cromwell.  

Once it finds info about a previous run to cache from, the `copy` vs `reference` option tells cromwell whether to copy the previous output files into a new location in the new execution directory, or just use the previous run outputs without creating a new copy of them.  At least that's my understanding, I've never actually used `reference`, so don't know if they're are any quirks in its behavior.

https://cromwell.readthedocs.io/en/stable/cromwell_features/CallCaching/